gusto/configuration.py

```diff
@@ -133,3 +134,25 @@ class CompressibleEadyParameters(CompressibleParameters, EadyParameters):
     theta_surf = 300.
     dthetady = theta_surf/g*EadyParameters.dbdy
     Pi0 = 0.0
+
+
+class AdvectionOptions(Configuration, metaclass=ABCMeta):
+
+    @abstractproperty
+    def name(self):
+        pass
+
```

Comment:
This is definitely a smarter and neater way of doing these options! Thanks so much for tidying up the mess I left behind...